core/src/main/scala/kafka/server/ZkAdminManager.scala

```diff
@@ -49,6 +49,7 @@ import org.apache.kafka.common.security.scram.internals.{ScramCredentialUtils, S
 import org.apache.kafka.common.utils.Sanitizer
 import org.apache.kafka.server.common.AdminOperationException
 import org.apache.kafka.server.config.{ConfigEntityName, ConfigType}
+import org.apache.kafka.server.config.KafkaConfig._
```

Comment:
I wonder if we should just import `KafkaConfig` and use `KAFKA.SOME_CONFIG` instead of importing all/statically KafkaConfig. WDYT?

Reply:
For this one and maybe other that import few config from KafkaConfig this would be nice improvement but some classes like DynamicBrokerConfig for example include a large set of config. I can update few to use  `KAFKA.SOME_CONFIG` if they need small set of config and leave the others specially that we when we move them to java we will do this anyway.

Reply:
Also at the moment we can't use `KafkaConfig` everywhere as there's another `KafkaConfig` in scala

Reply:
Ah right, that's a good point. Thanks

Reply:
After the last commit, this shouldn't be an issue anymore as the configs are in KafkaLogConfigs instead of KafkaConfig so there is no conflict anymore